frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-alerts.tsx

```diff
@@ -25,16 +22,32 @@ export const isGuestAgentVersionNotSupported = (vmi: VMIKind) => {
   return conditions && conditions.length > 0 && conditions[0].status === 'True';
 };
 
-export const VMAlerts: React.FC<VMAlertsProps> = ({ vmi }) => (
-  <AlertsBody>
-    {vmi && vmi.status && !isGuestAgentInstalled(vmi) && !isGuestAgentVersionNotSupported(vmi) && (
-      <StatusItem Icon={BlueInfoCircleIcon} message={NO_GUEST_AGENT_MESSAGE} />
-    )}
-    {vmi && vmi.status && isGuestAgentVersionNotSupported(vmi) && (
-      <StatusItem Icon={BlueInfoCircleIcon} message={GUEST_AGENT_VERSION_NOT_SUPPORTED_MESSAGE} />
-    )}
-  </AlertsBody>
-);
+export const VMAlerts: React.FC<VMAlertsProps> = ({ vmi }) => {
+  const { t } = useTranslation();
+  return (
+    <AlertsBody>
+      {vmi &&
```

Comment:
Can we extract these conditions into two variables?
```
const guestAgentNotSupported = vmi && vmi.status && isGuestAgentVersionNotSupported(vmi);
const guestAgentNotInstalled = vmi && vmi.status && !isGuestAgentInstalled(vmi) && !isGuestAgentVersionNotSupported(vmi);
```

It would be even cleaner if we use optional chaining.
```
const guestAgentNotSupported = vmi?.status && isGuestAgentVersionNotSupported(vmi);
const guestAgentNotInstalled = vmi?.status && !isGuestAgentInstalled(vmi) && !isGuestAgentVersionNotSupported(vmi);
```